io-engine/src/grpc/v0/json_grpc.rs

```diff
@@ -36,9 +36,12 @@ impl JsonRpc for JsonRpcSvc {
     ) -> GrpcResult<JsonRpcReply> {
         let args = request.into_inner();
 
-        let result = self
+        let result = match self
             .spdk_jsonrpc_call(&args.method, empty_as_none(&args.params))
-            .await?;
+            .await {
+                Ok(val) => val,
+                Err(_) => todo!(),
```

Comment:
@dsavitskiy @tiagolobocastro This line is modified to accommodate lack of implicit error conversion here coming up as a result of package version updates. But 'todo' doesn't look very correct here. Any suggestion to make it better?